cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Payment Card Industry Data Security Standard (PCI DSS) compliance in CockroachDB Cloud Dedicated
+summary: Learn about compliance with Payment Card Industry Data Security Standard (PCI DSS) for CockroachDB Cloud Dedicated clusters.
+toc: true
+docs_area: manage.security
+---
+
+The Payment Card Industry Data Security Standard (PCI DSS) is a series of requirements for the safe handling of sensitive data associated with credit and debit cards. In the rest of this article, this data is referred to as "card data". When implemented correctly, PCI DSS helps to protect your organization and your customers from fraud. PCI DSS is mandated by credit card issuers but administered by the [Payment Card Industry Security Standards Council](https://www.pcisecuritystandards.org/).
+
+{% include v22.1/dedicated-pci-compliance.md %}
+
+Responsibility for compliance with PCI DSS is shared among multiple parties, including card issuers, banks, software-as-a-service (SaaS) providers, and retail merchants. Compliance measures are implemented as a series of business practices, security controls, and technological solutions. An organization's compliance with PCI DSS is certified by a PCI Qualified Security Assessor (QSA).
+
+This article provides information about {{ site.data.products.dedicated }}'s compliance with PCI DSS, describes some of the ways that {{ site.data.products.db }} implements and enforces compliance, and illustrates some of the types of changes you may need to implement outside of your {{ site.data.products.dedicated }} clusters. For detailed instructions and working examples, refer to [Integrate CockroachDB Dedicated with Satori](/docs/{{site.versions["stable"]}}/satori-integration.html).
+
+## Overview of PCI DSS
+
+When a system complies with PCI DSS, the system meets a series of requirements, according to a PCI Qualified Security Assessor (QSA). For example:
+
+1. A firewall is installed and configured to scan all network traffic and prevent untrusted networks from accessing the system.
```

Comment:
Perhaps this list should be reproduced exactly and with a precise citation from the PCI DSS Quick Reference Guide, rather than the *very close* paraphrasing here.